packages/synapse-react-client/src/lib/containers/personal_access_token/CreateAccessTokenModal.tsx

```diff
@@ -85,118 +77,82 @@ export const CreateAccessTokenModal: React.FunctionComponent<
     }
   }
 
+  const dialogContent = showCreatedToken ? (
+    <>
+      <span className="SRC-boldText">
+        This token will not be able to be retrieved again.
+      </span>{' '}
+      <span>
+        If needed, generate a new personal access token, and delete this one.
+      </span>
+      <div className="AccessTokenModal__CopyToClipboardContainer">
+        <CopyToClipboardInput value={createdToken} inputWidth={'350px'} />
+      </div>
+      <p>
+        This token grants access to your account functions and should be treated
+        like a password.
+      </p>
+    </>
+  ) : (
+    <div className="SRC-marginFive">
+      <TextField
+        autoFocus
+        label="Token Name"
+        value={tokenName}
+        onChange={handleTokenNameChange}
+        placeholder="e.g. Synapse command line access on my laptop"
+      />
+      <div className="AccessTokenModal SRC-marginBottomTop">
+        <FormLabel className="SRC-boldText">Token Permissions</FormLabel>
```

Comment:
Replace this `FormLabel` with a MUI `Typography` and we can get rid of the `react-bootstrap` import in this file completely!

This Typography wouldn't need to be a `label` component, because it looks like this text isn't labeling any particular input.

Reply:
Done in the next commit!